red-canary/playbooks/playbooks.json

```
@@ -0,0 +1,889 @@
+            }
+          ],
+          "routes": [
+            {
+              "@type": "WorkflowRoute",
+              "name": "Alerts -> Get Endpoint",
+              "targetStep": "/api/3/workflow_steps/4b980ea9-3cef-4905-99df-8b4518e04f7a",
+              "sourceStep": "/api/3/workflow_steps/c88ee6fd-a59f-4c65-9c07-d34ffe06d42e",
+              "label": null,
+              "isExecuted": false,
+              "uuid": "b3fbb641-a72f-49eb-bd41-8e1c8650b167"
+            }
+          ],
+          "priority": null,
+          "uuid": "022e99de-ea76-46f5-889f-7b94476d51eb",
+          "recordTags": [
+            "RedCanary",
+            "red-canary"
+          ],
+          "id": 18092,
+          "createUser": "/api/3/appliances/57545210-2adc-472b-a24f-2df6ee8dfe63",
+          "createDate": 1649397930,
+          "modifyUser": "/api/3/people/3451141c-bac6-467c-8d72-85e0fab569ce",
+          "modifyDate": 1649400343,
+          "owners": [],
+          "isPrivate": false
+        },
+        {
+          "@type": "Workflow",
+          "triggerLimit": null,
+          "name": "Update Remediation State",
+          "aliasName": null,
+          "tag": "#Red Canary",
+          "description": "Update the remediation state of detection in Red Canary based on input parameters you have specified.",
+          "isActive": false,
+          "debug": false,
+          "singleRecordExecution": false,
+          "remoteExecutableFlag": false,
+          "parameters": [],
+          "synchronous": false,
+          "lastModifyDate": 1649400906,
+          "collection": "/api/3/workflow_collections/bb5b2635-0f73-4fde-a46b-ab6f7c22078a",
+          "versions": [],
+          "triggerStep": "/api/3/workflow_steps/ea5646a1-04fe-4497-b19b-4b79de5fbe3f",
+          "steps": [
+            {
+              "@type": "WorkflowStep",
+              "name": "Update Remediation State",
+              "description": null,
+              "arguments": {
+                "name": "Red Canary",
+                "config": "453f48ea-18a7-4a36-a9d0-ec899dafab71",
+                "params": {
+                  "comment": "",
+                  "detection_id": 37623,
+                  "remediation_state": "Not Remediated False Positive"
+                },
+                "version": "1.0.0",
+                "connector": "red-canary",
+                "operation": "update_remediation_state",
+                "operationTitle": "Update Remediation State",
+                "step_variables": {
+                  "output_data": "{{vars.result}}"
+                }
+              },
+              "status": null,
+              "top": "120",
+              "left": "188",
+              "stepType": "/api/3/workflow_step_types/0bfed618-0316-11e7-93ae-92361f002671",
+              "uuid": "da5fe1cf-2112-4212-a791-ef71a5447ac5",
+              "id": 60124
+            },
+            {
+              "@type": "WorkflowStep",
+              "name": "Alerts",
```

```suggestion
              "name": "Start",
```
make these changes to all PBs.